reporting_database_generator/00_Creation.sql

```diff
@@ -1,2 +1 @@
-create database `{{params.reporting_database}}` default character set=utf8mb4 default collate=utf8mb4_unicode_ci;
-drop database `{{params.last_reporting_database}}`;
\ No newline at end of file
+create database `PatentsView_{{ dag_run.logical_date | ds_nodash }}` default character set=utf8mb4 default collate=utf8mb4_unicode_ci;
```

Comment:
From my understanding the logical date/execution date would have the start of the interval, not end. Please confirm if this will work with whatever the start date of the DAG will be

Reply:
This is tested and should work correctly.